pkg/apis/batch/validation/validation.go

```diff
@@ -31,7 +31,7 @@ import (
 
 // TODO: generalize for other controller objects that will follow the same pattern, such as ReplicaSet and DaemonSet, and
 // move to new location.  Replace batch.Job with an interface.
-//
+
 // ValidateGeneratedSelector validates that the generated selector on a controller object match the controller object
 // metadata, and the labels on the pod template are as generated.
```

Comment:
What about this?

```
// ValidateGeneratedSelector validates that the generated selector on a controller object match the controller object
// metadata, and the labels on the pod template are as generated.	// metadata, and the labels on the pod template are as generated.
//
// TODO: generalize for other controller objects that will follow the same pattern, such as ReplicaSet and DaemonSet, and
// move to new location.  Replace batch.Job with an interface.	// move to new location.  Replace batch.Job with an interface.
```

Reply:
Thanks for the review @draveness! 🎉 
That makes more sense. I updated it.